core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -1220,6 +1220,9 @@ public function loadDetails()
 			$this->folderUrl = $folderUrl;
 		}
 
+		$container = System::getContainer();
+		$request = $container->get('request_stack')->getCurrentRequest();
```

Comment:
Now that we are using the request object, we can replace `Environment::get('path')` in line 1387 with `$request->getBasePath()`.

Reply:
The request object is in `loadDetails`. Line 1387 is `getFrontendUrl`. I can of course change it so that we get the request from the request stack there too instead of using `Contao\Environment`.

Reply:
You are right, it is unrelated. Never mind.